src/components/Layout/Sidebar.tsx

```diff
@@ -26,10 +26,13 @@ const renderItems = (
     {items.map((item, index) => {
       let isItemActive = false;
       let isItemCurrent = false;
+      let isItemNonProduct = false;
 
       if (isInternalItem(item)) {
         isItemActive = currentPath.startsWith(item.to);
```

Comment:
In the future, we may want to change this from startsWith to something like path[1] or wherever minfraud/geoip appears in the url path